### PR TITLE
Consensus snapshot cards layout

### PR DESCRIPTION
**Pull request by buberdds**
Screenshot from 2024-01-25 17-19-35

### CODE REVIEW

File: src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      query: {
+        cacheTime: 0,
+      },
+    },
+  )
+  const validators = validatorsQuery.data?.data.validators
+
+  return (
+    <SnapshotCard title={t('validator.listTitle')}>
+      {!!validators?.length && (
```
Comment (lukaw3d) [inline src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx:52]: Is this a replacement for loading state, or a guard for bad API?

File: src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  ]
+}
+
+export const SnapshotValidators: FC<{ scope: SearchScope }> = ({ scope }) => {
+  const { t } = useTranslation()
+  const { network } = scope
+
+  const validatorsQuery = useGetConsensusValidators(
+    network,
+    { limit: API_MAX_TOTAL_COUNT },
```
Comment (lukaw3d) [inline src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx:41]: (related: https://github.com/oasisprotocol/explorer/issues/226)

File: src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      activeCount++
+    } else {
+      inactiveCount++
+    }
+  })
+
+  return [
+    { label: t('validator.active'), value: activeCount },
+    { label: t('validator.inactive'), value: inactiveCount },
+  ]
```
Comment (lukaw3d) [inline src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx:32]: Note: we'll probably want more options https://github.com/oasisprotocol/oasis-wallet-web/issues/748

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx:52]: guard, but there is another one in `countValidatorsState` so I can kill this one

**Pull request merged**